core/modules/color/js/color.js

```diff
@@ -80,7 +80,7 @@ Backdrop.behaviors.color = {
     $('#system-theme-settings').addClass('has-preview').after(settings.colorPreviewMarkup);
     // Wait for the iframe to be loaded before attempting to apply the preview
     // settings for the first time.
-    $('#preview').load(function () {
+    $('#preview').on('load', function () {
```

Comment:
This is causing the preview to "flicker" (seems like it's reloading every second or so).

Reply:
I don't think this is the cause. When I switch to jQuery 1 it goes away.